pages/index.vue

```diff
@@ -568,14 +568,13 @@
             </div>
           </div>
           <div id="response-details-wrapper">
-            <pre>
-              <code
-  ref="responseBody"
-  id="body"
-  rows="16"
-  placeholder="(waiting to send request)"
->{{response.body}}</code>
-            </pre>
+            <ResponseBody
+              :value="responseBodyText"
+              theme="dracula"
```

Comment:
making `theme` a prop is better since, while changing theme to light mode, we could fire up a light theme for the editor 👍

Reply:
absolutely👍
which theme is good for light mode?
[http://ajaxorg.github.io/ace-builds/kitchen-sink.html](http://ajaxorg.github.io/ace-builds/kitchen-sink.html)

Reply:
XCode 😊

Reply:
I think there are three options
1. we can use the selected PW theme in setting and based on that choose the proper theme for ace editor:
- kanda dark -> dracula
- clearly white -> xcode
- just black -> ???
2. use theme prop for pass any ace available themes
3. both above options. use selected PW theme to define ace theme but if a theme prop exist prefer that instead PW theme.

what do you think?